docs/docsite/rst/community/collection_contributors/collection_requirements.rst

```diff
@@ -85,34 +85,36 @@ Python requirements for a collection vary between **controller environment** and
 
 One example scenario where the "even if" clause comes into play is when using cloud modules. These modules mostly run on the controller node but in some environments, the controller might run on one machine inside a demilitarized zone which cannot directly access the cloud machines. The user has to have the cloud modules run on a bastion host/jump server which has access to the cloud machines.
 
+An **eligible controller Python version** for a collection is a Python version that is supported on the controller side by at least one ansible-core version that the collection supports. Similarly, an **eligible target Python version** for a collection is a Python version that is supported on the target side by at least one ansible-core version that the collection supports. The eligible controller and target Python versions can be determined from the :ref:`ansible_core_support_matrix` and from the ``requires_ansible`` value in ``meta/runtime.yml`` in the collection.
+
 .. _coll_controller_req:
 
 Controller environment
 ~~~~~~~~~~~~~~~~~~~~~~
 
-In the controller environment, collections MUST support Python 2 (version 2.7) and Python 3 (Version 3.6 and higher), unless required libraries do not support these versions. Collections SHOULD also support Python v3.5 if all required libraries support this version.
+Collections MUST support all eligible controller Python versions in the controller environment, unless required libraries do not support these Python versions. The :ref:`Steering Committee <steering_responsibilities>` can grant other exceptions on a case-by-case basis. 
```

Comment:
Could this all be simplified to:
Collections MUST support all Python versions listed in ref`ansible_core_support_matrix` for all ansible-core versions that the collection supports. The exception is if  required libraries for the collection do not support these Python versions. Steering committee blablabla...

Then we avoid having to define 'eligible' python versions entirely have less for the reader to ...read and understand.
`

Reply:
I think it's too late to make larger changes to this, since we already started voting on it - see https://github.com/ansible-community/community-topics/discussions/268. Right now we should stick so simple wording changes or *small* formulation changes that don't change meanings.

Also if we simplify it, it should be

> Collections MUST support all Python versions listed in :ref:`ansible_core_support_matrix` in the Controller Python column for all ansible-core versions that the collection supports.

Also this needs to be repeated six times in the text (sometimes with Controller Python, sometimes with Target Python). I'm not sure that really improves readability.

Reply:
let's merge it as voted and then think about simplifications. I think the requirements (especially Python-related) look hard to read and understand overall (at least to me)